src/utils/hash/transactionHash/v3.ts

```diff
@@ -24,17 +24,33 @@ export function hashDAMode(nonceDAMode: BigNumberish, feeDAMode: BigNumberish) {
   return (BigInt(nonceDAMode) << DATA_AVAILABILITY_MODE_BITS) + BigInt(feeDAMode);
 }
 
-export function hashFeeField(tip: BigNumberish, bounds: ResourceBounds) {
-  const L1Bound =
+export function encodeResourceBoundsL1(bounds: ResourceBounds): bigint {
```

Comment:
maybe add desc, and note what this encoding is for

Reply:
Added.